src/syscheckd/db/src/dbFileItem.cpp

```diff
@@ -10,22 +10,26 @@
  */
 #include "dbFileItem.hpp"
 
-void FileItem::createFimEntry()
+void FileItem::createFimEntry(fim_entry& fim)
```

Comment:
Come back to before version and after my pull-request will be merged I am reviewing again these changes